tests/drum/test_fit.py

```diff
@@ -214,6 +211,56 @@ def test_fit(
             cmd, "Failed in {} command line! {}".format(ArgumentsOptions.MAIN_COMMAND, cmd)
         )
 
+    @pytest.mark.parametrize(
```

Comment:
it felt like these were starting to get different enough that it made sense to break it out, but I could be wrong